record-manager-design-list.component.ts

```
@@ -0,0 +1,1062 @@
+    }
+    if (isSelected == false) {
+      this.arrSelected.push({ 'Index': index, 'Row': row, 'Edit': 'edit' });
+    }
+    else {
+      this.arrSelected = this.arrSelected.filter(function (item) {
+        return item.Index !== index;
+      });
+    }
+  }
+
+  getCogIconColor(Array:any) {
```

Biome (1.9.4)

[error] Do not shadow the global "Array" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)